config.json

```diff
@@ -116,7 +116,8 @@
                     "name": "Customizing the Template"
                 },
                 {
-                    "name": "Customizing the Keyboard"
+                    "name": "Customizing the Keyboard",
+                    "platforms": ["iOS", "Android", "JavaEE", "JavaSE"]
```

Comment:
I think that this guide should still exist and just say that this manager isn't available yet, and to customize the keyboard they currently need to use the `SetGlobalProperties` RPC.